hugegraph-llm/src/hugegraph_llm/api/rag_api.py

```diff
@@ -126,3 +129,14 @@ def rerank_config_api(req: RerankerConfigRequest):
         else:
             res = status.HTTP_501_NOT_IMPLEMENTED
         return generate_response(RAGResponse(status_code=res, message="Missing Value"))
+
+    @router.post("/logs", status_code=status.HTTP_200_OK)
```

Comment:
prefer to separate it in a single api file (like `util_api.py` or `admin_api.py`)